pkg/storage/pebble.go

```diff
@@ -2507,14 +2507,14 @@ func (p *Pebble) CreateCheckpoint(dir string, spans []roachpb.Span) error {
 // version associated with it, since they did so during the fence version.
 var pebbleFormatVersionMap = map[clusterversion.Key]pebble.FormatMajorVersion{
 	clusterversion.V25_4_PebbleFormatV2BlobFiles: pebble.FormatV2BlobFiles,
-	clusterversion.V25_3:                         pebble.FormatValueSeparation,
+	clusterversion.V25_4:                         pebble.FormatMarkForCompactionInVersionEdit,
```

Comment:
I set V25_4 to the [latest version in format_major_version.go](https://github.com/cockroachdb/pebble/blob/master/format_major_version.go#L254)

@RaduBerinde can you confirm:
1. this is the correct value to set for V25_4?
2. that we want this to happen as part of [M.1: Bump Current Version (Master Branch)](https://github.com/cockroachdb/cockroach/blob/master/pkg/clusterversion/README.md#m1-bump-current-version)? (if so I'll add this to the runbook)
3. [question] should we also do this as part of [R.2: Mint release](https://github.com/cockroachdb/cockroach/blob/master/pkg/clusterversion/README.md#r2-mint-release)?

Reply:
No, we should just remove the V25_3 entry. We can change V25_4_PebbleFormatV2BlobFiles to V25_4, or we can leave it for later (when we clean up deprecated gates). In any case, we'd only have one entry with `FormatV2BlobFiles`.